lib/OBSWebSocket.js

```diff
@@ -37,20 +35,20 @@ class OBSWebSocket extends Socket {
       const messageId = args['message-id'] = generateMessageId(); // eslint-disable-line no-multi-assign
 
       // Submit the request to the websocket.
-      log.debug('[Core:send]', messageId, requestType, args);
+      debug('[send] %s %s %O', messageId, requestType, args);
       this._socket.send(JSON.stringify(args));
```

Comment:
Also just noticed that you don't get proper promise rejection when sending a request while not connected to a websocket.